logging/lazy_fields.go

```diff
@@ -0,0 +1,179 @@
+package logging
+
+import (
+	"fmt"
+	"net"
+	"net/url"
+	"strconv"
+	"strings"
+	"time"
+
+	spec "github.com/attestantio/go-eth2-client/spec/phase0"
+	specqbft "github.com/bloxapp/ssv-spec/qbft"
+	spectypes "github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv/protocol/v2/blockchain/beacon"
+	protocolp2p "github.com/bloxapp/ssv/protocol/v2/p2p"
+	"github.com/dgraph-io/ristretto"
+
+	"github.com/libp2p/go-libp2p/core/peer"
+
+	"github.com/bloxapp/ssv/network/records"
+	"github.com/ethereum/go-ethereum/p2p/enode"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+const (
```

Comment:
Can we flip the names to start with `Field`?  I believe it's Go-ish & also good for autocomplete
```go
FieldAddress             = "address"
FieldBindIP              = "bindIP"
FieldBlockCacheMetrics   = "blockCacheMetricsField"
FieldCurrentSlot         = "currentSlot"
FieldDuration            = "duration"
FieldEnr                 = "enr"
FieldEventID             = "eventID"
FieldFromBlock           = "fromBlock"
FieldIdentifier          = "identifier"
FieldIndexCacheMetrics   = "indexCacheMetrics"
FieldMessageID           = "messageID"
FieldPeerID              = "peerID"
FieldPrivateKey          = "privKey"
FieldPubKey              = "pubKey"
FieldResults             = "results"
FieldStartTime           = "startTime"
FieldSubnets             = "subnets"
FieldSyncOffset          = "syncOffset"
FieldTargetNodeEnr       = "targetNodeEnr"
FieldTxHash              = "txHash"
FieldUpdatedEnrLocalNode = "updated_enr"
```